package.json

```diff
@@ -13,7 +13,7 @@
     "./dist/formio.core.min.js": "./dist/formio.core.min.js"
   },
   "scripts": {
-    "test": "TEST=1 mocha -r ts-node/register -r tsconfig-paths/register -r mock-local-storage -r jsdom-global/register -b -t 0 'src/**/__tests__/*.test.ts'",
+    "test": "TEST=1 nyc --reporter=lcov --reporter=text --reporter=text-summary mocha -r ts-node/register -r tsconfig-paths/register -r mock-local-storage -r jsdom-global/register -t 0 'src/**/__tests__/*.test.ts'",
```

Comment:
This will run the test through the code coverage.  it creates an HTML report to look at manually and file coverage and summary reports in the console.